go.cfg

```diff
@@ -6,7 +6,7 @@
 cfg_fnDatabase: godb.pickle
 
 # F5's favicon for use in client browsers
-cfg_urlFavicon: http://www.f5.com/favicon.ico
+cfg_urlFavicon: https://www.f5.com/favicon.ico
```

Comment:
It always seemed weird this was set to a URL for F5's web server. We might consider just storing a temporary icon locally and using that instead of having this external dependency.